integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -10258,6 +10258,10 @@ msgid "An error has occurred. Please contact an administrator."
 msgstr ""
 "Ein Fehler ist aufgetreten. Bitte kontaktieren Sie eine:n Administrator:in."
 
+#: cms/views/pages/page_permission_actions.py
+msgid "Inactive users cannot be assigned to pages."
+msgstr "Benutzer:innen können keiner Seite zugeordnet werden, solange ihr Nutzerkonto nicht aktiviert wurde."
```

Comment:
```suggestion
msgstr "Benutzer:innen können keiner Seite zugeordnet werden, solange ihr Konto nicht aktiviert wurde."
```
Super nit pick: I think we usually use the gender neutral version and just say "Konto"

Reply:
Thanks for the suggestion, I changed it :)